code/modules/modular_computers/file_system/picture_file.dm

```diff
@@ -0,0 +1,39 @@
+/**
+ * PNG file type
+ * Stores a picture to be used for programs.
+ */
```

Comment:
This documentation needs to be next to the *type*. Right now you are documenting the global variable `next_ntos_picture_uid`.